model/geo/registry.yaml

```diff
@@ -56,18 +56,12 @@ groups:
         brief: >
           Two-letter ISO Country Code ([ISO 3166-1 alpha2](https://wikipedia.org/wiki/ISO_3166-1#Codes)).
         examples: [ 'CA' ]
-      - id: geo.location.lon
+      - id: geo.location
         stability: experimental
-        type: double
+        type: double[]
         brief: >
-          Longitude of the geo location in [WGS84](https://wikipedia.org/wiki/World_Geodetic_System#WGS84).
-        examples: [ -73.614830 ]
-      - id: geo.location.lat
-        stability: experimental
-        type: double
-        brief: >
-          Latitude of the geo location in [WGS84](https://wikipedia.org/wiki/World_Geodetic_System#WGS84).
-        examples: [ 45.505918 ]
+          Latitude and longitude (in that order) of the geo location in [WGS84](https://wikipedia.org/wiki/World_Geodetic_System#WGS84).
```

Comment:
Ordering is based on ChatGPT answer:

The standard order for storing geographic coordinates in an array often depends on the context and the conventions of the tools you're using:

1. [Latitude, Longitude] ([lat, lon]):
- Common in general programming contexts, especially APIs and services like Google Maps.
- Latitude specifies the north-south position, and longitude specifies the east-west position.
2. [Longitude, Latitude] ([lon, lat]):
- Used by geospatial standards like GeoJSON.
- GeoJSON adheres to the right-hand rule, with longitude (x-axis) first, followed by latitude (y-axis).

Recommendation:
- Use [lat, lon] for most general applications and mapping tools.
- Use [lon, lat] for GeoJSON and geospatial libraries like Mapbox or Leaflet.

Reply:
FYI Elasticsearch uses [lon,lat]: https://www.elastic.co/guide/en/elasticsearch/reference/current/geo-point.html

Reply:
I think it makes sense to align with [GeoJSON](https://geojson.org/) and [Well-Known Text](https://docs.opengeospatial.org/is/12-063r5/12-063r5.html) and order them like `[lon,lat]`.

Reply:
Ok, changed it to `[lon,lat]`.

I personally don't really have an opinion, only reason I originally proposed `[lat,lon]` was that I felt SemConv is closer to a "general programming context" than to a geo standard.

Let's go with `[lon,lat]` now.